src/meta/src/rpc/server.rs

```diff
@@ -143,6 +142,17 @@ pub async fn rpc_serve_with_store<S: MetaStore>(
             let (leader_info, is_leader) = note_status_leader_rx.borrow().clone();
             let leader_addr = leader_info_to_host_addr(leader_info);
 
+            // Implementation of naive fencing mechanism:
+            // leader nodes should panic if they loose their leader position
+            // Current implementation panics, if leader looses lease.
+            // Alternative implementation that panics only if leader is unable to re-acquire lease:
+            // if was_leader && !is_leader {
```

Comment:
Please determine if this is required, or just remove the debug comment

Reply:
I talked with @yezizp2012 about this. It is no longer required.